pyproject.toml

```diff
@@ -10,7 +10,8 @@ license = "MIT"
 
 authors = [
     "Umit Buyuksahin <umit.buyuksahin@exasol.com>",
-    "Torsten Kilias <torsten.kilias@exasol.com>"
+    "Torsten Kilias <torsten.kilias@exasol.com>",
```

Comment:
Should we keep old authors?

Reply:
Imho, yes but we could explicitly set `maintainers` field to the current maintainers (also will be shown on pypi)